File: ticketProject5-DistributedC/TicketSystemCC/networking/BuyerDTO.cs
using System;

namespace networking.dto
{

	[Serializable]
	public class BuyerDTO
	{
		private String idAR;
		private String firstName;
		private String lastName;
		private String noTickets;


		public BuyerDTO(String idAR, String firstName, String lastName, String noTickets)
		{
			this.idAR = idAR;
			this.firstName = firstName;
			this.lastName = lastName;
			this.noTickets = noTickets;
		}

		public virtual string IdAR
		{
			get
			{
				return idAR;
			}
			set
			{
				this.idAR = value;
			}
		}

		public virtual string FirstName
		{
			get
			{
				return firstName;
			}
			set
			{
				this.firstName = value;
			}
		}

		public virtual string LastName
		{
			get
			{
				return lastName;
			}
			set
			{
				this.lastName = value;
			}
		}

		public virtual string NoTickets
		{
			get
			{
				return noTickets;
			}
			set
			{
				this.noTickets = value;
			}
		}


	}


}

File: ticketProject9-REST/TS-REST/TicketNetworking/src/main/java/ticket/network/dto/BuyerDTO.java
package ticket.network.dto;

import java.io.Serializable;


public class BuyerDTO implements Serializable{
    private String idAR;
    private String firstName;
    private String lastName;
    private String noTickets;

    public BuyerDTO(String idAR, String firstName, String lastName, String noTickets) {
        this.idAR = idAR;
        this.firstName = firstName;
        this.lastName = lastName;
        this.noTickets = noTickets;
    }

    public String getARId() {
        return idAR;
    }
    public String getFirstName() {
        return firstName;
    }
    public String getLastName() { return lastName; }
    public String getNoTickets() {
        return noTickets;
    }

    public void setIdAR(String idAR) { this.idAR = idAR; }
    public void setFirstName(String firstName) { this.firstName = firstName; }
    public void setLastName(String lastName) { this.lastName = lastName; }
    public void setNoTickets(String noTickets) { this.noTickets = noTickets; }

    @Override
    public String toString(){
        return "BuyerDTO[" + idAR + " ; " + firstName + " ; " + lastName + " ; " + noTickets +"]";
    }
}
